lib/main.dart

```diff
@@ -1,5 +1,5 @@
 import 'package:flutter/material.dart';
-import 'package:paintroid/ui/theme.dart';
+import 'package:paintroid/ui/color_schemes.g.dart';
```

Comment:
Rename the file to `color_schemes.dart` as it might be ignored. In the future, we might use a package that generates files (using build_runner) with the extension `*.g.dart`. We generate all the files in the CI again, so the `*.g.dart` pattern will be added to .gitignore, and `color_schemes.g.dart` will get ignored.